Source/WorldActorRegistry/Private/WARWorldActorRegistrationComponent.cpp

```diff
@@ -0,0 +1,46 @@
+#include "WorldActorRegistry/Public/WARWorldActorRegistrationComponent.h"

```

Comment:
Only "WorldActorRegistry/Public/WARWorldActorRegistrationComponent.h"

you never have to have the Public folder in the include path

Reply:
Done.